docs/guides/server/caching.adoc

```diff
@@ -304,6 +304,18 @@ It requires a keystore with the certificate to use: `cache-embedded-mtls-key-sto
 The truststore contains the valid certificates to accept connection from, and it can be configured with `cache-embedded-mtls-trust-store-file` (path to the truststore), and `cache-embedded-mtls-trust-store-password` (password to decrypt it).
 To restrict unauthorized access, use a self-signed certificate for each {project_name} deployment.
 
+[NOTE]
+====
+**Zero Configuration Encryption**
+
+{project_name} offers a zero-configuration approach to encrypting network communication between nodes.
+This feature automatically generates self-signed certificates, eliminating the need for manual certificate creation and management.
```

Comment:
You should mention when it's set for the main HTTP server, it's inherited, right?

Reply:
This keys/certificate is just for the JGroups internal cache communication.